src/settings-ui/Settings.UI/SettingsXAML/Controls/SettingsPageControl/SettingsPageControl.xaml

```diff
@@ -19,23 +19,24 @@
             TrueValue="Visible" />
     </UserControl.Resources>
 
-    <Grid Padding="20,0,0,0" RowSpacing="24">
+    <Grid RowSpacing="16">
         <Grid.RowDefinitions>
             <RowDefinition Height="Auto" />
             <RowDefinition Height="*" />
         </Grid.RowDefinitions>
         <TextBlock
             x:Name="Header"
             MaxWidth="{StaticResource PageMaxWidth}"
+            Padding="36,0"
             HorizontalAlignment="Stretch"
             VerticalAlignment="Stretch"
             AutomationProperties.HeadingLevel="1"
             Style="{StaticResource TitleTextBlockStyle}"
             Text="{x:Bind ModuleTitle}" />
 
-        <ScrollViewer Grid.Row="1">
+        <ScrollViewer Grid.Row="1" Padding="36,0">
```

Comment:
This might be due to these changes: on smaller sizes we want to decrease the spacing on the left / right to show more content:

Top is this PR, bottom is `main`
![image](https://github.com/microsoft/PowerToys/assets/9866362/7aae3e1e-912e-405b-ab67-ea0234b3114b)

Reply:
> on smaller sizes we want to decrease the spacing on the left / right

Do you mean via a VisualState? I can look at that. Or simply decrease the "static" value? What are the best numbers then?

Reply:
I'm not sure if we want yet another visual state for this.. it was already working on `main` right? So maybe revert some of the padding changes you made - I think the values are the same but they now are e.g. margins instead of paddings

Reply:
I reset the basic margin to 24. Check out the latest commit to see the VidualState. 🙂

Reply:
It works, nice 👍! 2 things:

Are you using 12 here? Could be that 16 is more appropriate. That would align the text with the navview button and would make the scrollbar not overlap the card itself?
![image](https://github.com/microsoft/PowerToys/assets/9866362/19c0f3c8-9584-4a85-acb9-0e92229f5cb4)

There seems to be a difference between `main` in this PR with the gap between the window border and the content?
![image](https://github.com/microsoft/PowerToys/assets/9866362/2c81dbdf-dce6-4e79-ba64-e2ba697b3599)

Reply:
I'll look again

Reply:
> That would align the text with the navview button and would make the scrollbar not overlap the card itself?

Yes, the scrollbar is a good argument. But Windows Settings does not align titlebar buttons with text/content below it. It uses 24px.